Formula/pyqt-builder.rb

```diff
@@ -0,0 +1,18 @@
+class PyqtBuilder < Formula
+  desc "Tool to build PyQt"
+  homepage "https://www.riverbankcomputing.com/software/pyqt-builder/intro"
+  url "https://files.pythonhosted.org/packages/22/61/ddb67823c67ba12597908221413d817602a511e0c60fe926b58d49222332/PyQt-builder-1.9.0.tar.gz"
+  sha256 "54520b4d40d6e6d1307f9cebab4730ae78e01873c88866aa1e5192c24801005a"
+  license any_of: ["GPL-2.0-only", "GPL-3.0-only"]
+  head "https://www.riverbankcomputing.com/hg/PyQt-builder", using: :hg
+
+  depends_on "python@3.9"
+
+  def install
+    system "python3", *Language::Python.setup_install_args(prefix)
+  end
+
+  test do
+    system bin/"pyqt-bundle", "-V"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula